### ISSUE TITLE
[Feature discuss] Need we define a extend relationship for the configs which are same in all namespace?

### ISSUE THREAD

**Issue by brothelul** [alibaba/nacos#5535]
**Is your feature request related to a problem? Please describe.**

In our bussiness using of the config center, we usually have more than one namespace like dev,test,prod and other. maybe many of our config are same in each namespace. but Nacos donot have a parent or extend relationship of the config between all namespace. so we need create the same config in each namespace, it isn't friendly for use.

**Describe the solution you'd like**
introducing the concept of extend or parent for config. 

**Describe alternatives you've considered**
-

**Additional context**
-

**Comment by KomachiSion** [comment]
So you want implement a feature like a template, and change this template will take effect in all sub-config?

**Comment by haoyann** [comment]
I think there should be several key points to realize this feature.
1. sub-config can can override parent-config property.
2. sub-config can extends multiple parent-config.
3. to be added.

When creating a new project, datasource, rpc and other configs all extend parent-config, this facilitates configuration management and reduces duplicate configuration.

**Comment by brothelul** [comment]
> I think there should be several key points to realize this feature.
> 
> 1. sub-config can can override parent-config property.
> 2. sub-config can extends multiple parent-config.
> 3. to be added.
> 
> When creating a new project, datasource, rpc and other configs all extend parent-config, this facilitates configuration management and reduces duplicate configuration.

Yes, the scene is like this. but I don't get the point of No.2. `sub-config can extends multiple parent-config`.
